packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -459,6 +487,7 @@ export default class TestConfiguration {
 
     const headers: any = {
       Accept: "application/json",
+      Cookie: "",
```

Comment:
How come this is necessary?

Reply:
This just makes sure the cookie header is definitely overridden to be nothing, if its being mixed with some other headers.